unstructured/ingest/main.py

```diff
@@ -290,13 +290,13 @@ def main(
     if s3_url:
         doc_connector = S3Connector(
             config=SimpleS3Config(
+                path=s3_url,
+                access_kwargs={"anon": s3_anonymous},
                 download_dir=download_dir,
-                s3_url=s3_url,
                 output_dir=structured_output_dir,
-                # set to False to use your AWS creds (not needed for this public s3 url)
-                anonymous=s3_anonymous,
                 re_download=re_download,
                 preserve_downloads=preserve_downloads,
+                verbose=verbose,
```

Comment:
```suggestion
                verbose=verbose,
```
This should no longer be necessary, the verbosity should all be handled in `main.py` since the logger changes.

Reply:
Nice catch, probably there from a merge conflict resolved badly? I'll check the rest in case there's any other `verbose` still used :)

Reply:
@alvarobartt , I just removed that `verbose` line before seeing your comment. The s3 connector at least runs fine after that.